docker-compose.ytt.min.yaml

```diff
@@ -207,6 +207,218 @@ services:
       - rabbit
       - redis
 
+
+  onnx_bert_worker:
```

Comment:
should these services really be part of the "minimal" version of square?

Reply:
We copied the dependencies from the tasb_worker. When I try to remove rabbit or redis from any of our services, I get the following errors during requests:
- amqp.exceptions.AccessRefused: (0, 0): (403) ACCESS_REFUSED - Login was refused using authentication mechanism PLAIN. For details see the broker logfile.
- redis.exceptions.ResponseError: WRONGPASS invalid username-password pair or user is disabled.

I think that 'minimal" in this case refers to the UI not being included
- Maybe we should rename the file to "docker-compose.ytt.no-frontend.yaml"

Reply:
Right, services like Rabbit, Redis, Mongo etc they are dependencies of other core services, so no need to remove them.

However, I was more talking about the additional models that are added here. IMHO, this file should just deploy a single model and single datastore. This is basically just a demo of the each component. Therefore, I think we do not need additional models in this file. But please pitch in @Rachneet or @kwang2049 if you have a different opinion.

Reply:
@Rachneet do you want these model services to be added to the docker-compose.ytt.yaml or should they be deployed dynamically?

Reply:
They should be deployed dynamically. For testing purposes only, these services can be added to docker-compose.models.yaml file.